.github/workflows/checks.yaml

```diff
@@ -43,7 +43,6 @@ jobs:
       - name: Download go dependencies
         run: |
           go mod download
-          go mod download -modfile internal/tools/go.mod
```

Comment:
For my own understanding, why is this removed instead of changed to the new path?